setup.py

```diff
@@ -1,39 +1,43 @@
 #!/usr/bin/env python
-
 from os.path import abspath, join, dirname
 from platform import system
 from shutil import copyfile
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
+from setuptools import setup
 
-ccmscript = 'ccm'
+ccm_script = "ccm"
 if system() == "Windows":
-    copyfile('ccm', 'ccm.py')
-    ccmscript = 'ccm.py'
+    copyfile("ccm", "ccm.py")
+    ccm_script = "ccm.py"
+
+with open("requirements.txt") as requirements_file:
+    requires = [package for package in requirements_file.readlines()]
+
+with open(abspath(join(dirname(__file__), "README.md"))) as readme_file:
+    long_description = readme_file.read()
 
 setup(
-    name='ccm',
-    version='2.0.5',
-    description='Cassandra Cluster Manager',
-    long_description=open(abspath(join(dirname(__file__), 'README.md'))).read(),
-    author='Sylvain Lebresne',
-    author_email='sylvain@datastax.com',
-    url='https://github.com/pcmanus/ccm',
-    packages=['ccmlib', 'ccmlib.cmds'],
-    scripts=[ccmscript],
-    install_requires=['pyYaml', 'psutil', 'six >=1.4.1', 'requests'],
+    name="ccm",
+    version="2.0.5",
```

Comment:
if you are touching it, we should maybe manage the version by some git command, or a version file?